src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/ContextConfigurator.php

```diff
@@ -422,4 +422,14 @@ protected function addPaginationConfig()
 
         $this->configuration->offsetSetByPath($this->getSourcePath(self::PRODUCTS_PER_PAGE), $value);
     }
+
+    /**
+     * Inject requested _per_page parameters in the datagrid configuration
+     */
+    protected function addSearchAfter()
+    {
+        $identifier = $this->requestParams->get('search_after', null);
```

Comment:
Ok weird as it comes from request, but I get why you need to set a null value then